docs/content/how-to/creating-catalogs.md

```diff
@@ -175,3 +176,43 @@ Using the table option facilitates the convenient definition of Hive table param
 Parameters prefixed with `hive.` will be automatically defined in the `TBLPROPERTIES` of the Hive table. 
 For instance, using the option `hive.table.owner=Jon` will automatically add the parameter `table.owner=Jon` to the table properties during the creation process.
 
+## Creating a Catalog with JDBC Metastore
+
+By using the Paimon JDBC catalog, changes to the catalog will be directly stored in relational databases such as MySQL, postgres, etc.
+
+{{< tabs "jdbc-metastore-example" >}}
+
+{{< tab "Flink" >}}
+
+Paimon JDBC Catalog in Flink needs to correctly add the corresponding jar package for connecting to the database. You should first download JDBC  connector bundled jar and add it to classpath. such as MySQL, postgres
+
+| database type | Bundle Name          | SQL Client JAR                                                             |
+|:--------------|:---------------------|:---------------------------------------------------------------------------|
+| mysql         | mysql-connector-java | [Download](https://mvnrepository.com/artifact/mysql/mysql-connector-java)  |
+| postgres      | postgresql           | [Download](https://mvnrepository.com/artifact/org.postgresql/postgresql)   |
+
+```sql
+CREATE CATALOG my_jdbc WITH (
+    'type' = 'paimon',
+    'metastore' = 'jdbc',
+    'uri' = 'jdbc:mysql://<host>:<port>/<databaseName>',
+    'jdbc.user' = '...', 
+    'jdbc.password' = '...', 
+    'catalog-name'='jdbc'
+    'initialize-catalog-tables'='true'
+    'warehouse' = 'hdfs:///path/to/warehouse'
+);
+
+USE CATALOG my_jdbc;
+```
+You can define any connection parameters for a database with the prefix "jdbc.".
+
+You can define the "initialize-catalog-tables" configuration to automatically create tables required for the initial JdbcCatalog. If it is true, tables will be automatically created when initializing JdbcCatalog. If it is false, tables will not be automatically created and you must manually create them.
```

Comment:
> If it is false, tables will not be automatically created and you must manually create them.

Why is there such a need? I don't understand.

Reply:
> Why is there such a need? I don't understand.

Some online business databases do not allow table creation through jdbc and must be submitted for approval in advance

Reply:
The final decision is not to keep this configuration. It is recommended that users enable automatic table creation permissions for independent databases, which would be better

Reply:
> Some online business databases do not allow table creation through jdbc and must be submitted for approval in advance

You can document this.

Reply:
> > Some online business databases do not allow table creation through jdbc and must be submitted for approval in advance
> 
> You can document this.

Most users may not need it, I will remove this parameter and add it back if there is a strong requirement.